### PR TITLE
Make Empty fileds as red border after clicking add button

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
Don't close the PR. Ask me once for changes.

@tarunsamanta2k20 

video link:

image

https://user-images.githubusercontent.com/55488549/227728240-ab054ce8-1976-4bec-8e9f-3fb5231fa1ed.mp4

Issue link: #96 
Issue Description:
One minor issue I found while adding a new database connection is, If any field is kept empty, it does show "cannot be empty," which is good.
We can make the border of the empty field Red if it is empty. (Warning)

### CODE REVIEW

**Comment by paraswaykole** [comment]
There are conflicts! Please resolve the conflicts

**Comment by paraswaykole** [comment]
Is it possible to show the errors when the user presses the add button but before hitting the API? can you make that change?

**Comment by paraswaykole** [comment]
No! I mean't displaying the empty field as red only after pressing add button and not before that. By default it should be default input css style.

**Comment by tarunsamanta2k20** [comment]
@paraswaykole did you watch my video from conversation section? 

When I am pressing add but button then only red borders are getting visible. Right?

**Comment by paraswaykole** [comment]
Ok! The video doesn't play. I will test it and merge it then.

**Comment by paraswaykole** [comment]
Looks good! One last change. backend only throws `cannot be empty` error when "display name or host or port or database name" is empty. Can you add these checks only for these 4 fields now.

**Comment by tarunsamanta2k20** [comment]
@paraswaykole at the place of cannot be empty?

**Comment by tarunsamanta2k20** [comment]
> Looks good! One last change. backend only throws `cannot be empty` error when "display name or host or port or database name" is empty. Can you add these checks only for these 4 fields now?

@paraswaykole at the place of cannot be empty?

**Comment by paraswaykole** [comment]
Nope. For showing the red input boxes

**Comment by tarunsamanta2k20** [comment]
> Nope. For showing the red input boxes

@paraswaykole where do i have to add?

**Comment by paraswaykole** [comment]
You have already added the red border on the input boxes for all the input boxes when it throws error. Show the red border only on the 4 mentioned input boxes, remove the code for rest of them. Please make these changes.

four input boxes where red border should show when it throws errors are:
- display name
- host
- port
- database name

**Comment by tarunsamanta2k20** [comment]
> You have already added the red border on the input boxes for all the input boxes when it throws error. Show the red border only on the 4 mentioned input boxes, remove the code for rest of them. Please make these changes.
> 
> four input boxes where red border should show when it throws errors are:
> 
> * display name
> * host
> * port
> * database name

@paraswaykole what about checkboxes red border?

**Comment by paraswaykole** [comment]
> > You have already added the red border on the input boxes for all the input boxes when it throws error. Show the red border only on the 4 mentioned input boxes, remove the code for rest of them. Please make these changes.
> > four input boxes where red border should show when it throws errors are:
> > 
> > * display name
> > * host
> > * port
> > * database name
> 
> @paraswaykole what about checkboxes red border?

Not needed!

**Comment by tarunsamanta2k20** [comment]
image

**Comment by tarunsamanta2k20** [comment]
@paraswaykole done

File: frontend/src/pages/project/newdb.tsx [hunk 1]
```diff
@@ -100,14 +130,15 @@ const NewDBPage: FunctionComponent<{}> = () => {
                     name='dbName'
                     value={data.dbName}
                     onChange={e => handleChange(e)}
-                    placeholder="Enter a display name for database"
+                    placeholder="Enter a me for database"
```
Comment (paraswaykole) [inline frontend/src/pages/project/newdb.tsx:133]: Please revert back to the text "Enter a display name for database"

File: frontend/src/pages/project/newdb.tsx [hunk 2]
```diff
@@ -100,14 +130,15 @@ const NewDBPage: FunctionComponent<{}> = () => {
...
+                    style={inputError.error_1 ? inputStyle :  normal} 
                 />
                 <div className="field">
                     <label className="label">Database Type:</label>
                     <div className="control">
                         <div className="select">
-                            <select name="dbType" onChange={(e: React.ChangeEvent<HTMLSelectElement>) => { setData((prev) => ({ ...prev, [e.target.name]: e.target.value, dbScheme: "" })) }}>
-                                <option value={DBConnType.POSTGRES}>PostgreSQL</option>
+                            <select name="dbType" style= {normal} onChange={(e: React.ChangeEvent<HTMLSelectElement>) => { setData((prev)=> ({...prev, [e.target.name]:e.target.value, dbScheme :""}))}}>
+                                <option value={DBConnType.POSTGRES}>PostgresSQL</option>
```
Comment (paraswaykole) [inline frontend/src/pages/project/newdb.tsx:141]: Revert to "PostgreSQL"

**Comment by tarunsamanta2k20** [comment]
@paraswaykole please add me as a member of your organization. I wanna contribute more in tte future.

**Pull request merged**